tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -2061,11 +2063,20 @@ TEST_F(CompilerTest, CompileSameFilenameAndEntryThenTestPdbUtilsArgs) {
   CComPtr<IDxcCompiler> pCompiler;
   VERIFY_SUCCEEDED(CreateCompiler(&pCompiler));
 
+// Use UTF8 for linux.
+#ifdef _WIN32
   std::wstring shader = LR"x(
     [RootSignature("")] float PSMain() : SV_Target {
       return 0;
     }
   )x";
+#else
+  std::string shader = R"x(
+    [RootSignature("")] float PSMain() : SV_Target  {
+      return 0;
+    }
+  )x";
+#endif
```

Comment:
I think a simpler solution would be to just use UTF8 for both platforms. The wide char isn't really the point of this test.

Reply:
Updated.